.gitignore

```diff
@@ -4,6 +4,8 @@ assets/seed
 
 # Built binaries
 build/bin
+verifier.test
```

Comment:
where is this binary from?

Reply:
I think it's generated by `common/libzkp/e2e-test.sh`.

Reply:
is this file used? since core/prover_test.go is invalid now

Reply:
Resolve this and talk in another thread